crates/typst-library/src/text/raw.rs

```diff
@@ -56,6 +62,35 @@ use crate::World;
 /// #raw("fn " + "main() {}", lang: "rust")
 /// ```
 ///
+/// # Styling
+///
+/// By default, the `raw` element uses the `DejaVu Sans Mono` font, available
+/// by default in Typst, with a smaller font size of `0.8em` (that is, 80% of
+/// the global font size). You can customize these properties with a show-set
+/// rule:
+///
+/// ````example
+/// // Switch to Cascadia Code for both inline and block raw.
```

Comment:
The example is a bit wide so that it won't fit side-by-side, which is somewhat unfortunate. But the show-set rules would probably need to be broken in a somewhat unreadable way to make it fit into the 38 char limit.

Reply:
Yeah, this seems a bit unavoidable, as otherwise we'd be encouraging people copying the example to use those otherwise unnecessary line length workarounds.

Reply:
I've broken up the comment, but I don't think I can improve it further without using additional variable definitions for the fonts and element functions.